es_ES/_data/catalog/widgets.json

```diff
@@ -0,0 +1,1582 @@
+    "categories": [
+      "Animation and Motion"
+    ],
+    "subcategories": [
+      "Routing"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Hero-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "AnimatedBuilder",
+    "description": "Un widget de propósito general para construir animaciones. AnimatedBuilder es útil para widgets más complejos que deseen incluir una animación como parte de una función de construcción más grande. Para usar AnimatedBuilder, simplemente construya el widget y pásele una función de constructor.",
+    "categories": [
+      "Animation and Motion"
+    ],
+    "subcategories": [
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/AnimatedBuilder-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "Draggable",
+    "description": "Un widget que se puede arrastrar desde un DragTarget. Cuando un widget arrastrable reconoce el inicio de un gesto de arrastre, muestra un widget de retroalimentación que rastrea el dedo del usuario a través de la pantalla. Si el usuario levanta el dedo mientras está en la parte superior de un DragTarget, se le da la oportunidad de aceptar los datos transportados por el objetivo arrastrable.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/Draggable-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "LongPressDraggable",
+    "description": "Hace que su hijo sea arrastrable a partir de una pulsación larga.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/LongPressDraggable-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "GestureDetector",
+    "description": "Un widget que detecta gestos. Intenta reconocer gestos que corresponden a sus llamadas de retorno no nulas. Si este widget tiene un hijo, se remite a ese hijo por su comportamiento de tamaño. Si no tiene un hijo, crece para adaptarse a los padres.",
+    "categories": [
+    ],
+    "subcategories": [
+      "Touch interactions"
+    ],
+    "link": "https://docs.flutter.io/flutter/widgets/GestureDetector-class.html",
+    "image": "<img alt='' src='/images/catalog-widget-placeholder.png'>"
+  },
+  {
+    "name": "DragTarget",
+    "description": "Un widget que recibe datos cuando se suelta un widget arrastrable. Cuando se arrastra un arrastrable encima de un objetivo de arrastre, se pregunta al objetivo de arrastre si aceptará los datos que lleva el arrastrable. Si el usuario suelta el arrastrable encima del objetivo de arrastre (y el objetivo de arrastre ha indicado que aceptará los datos del arrastrable), se le pide al objetivo de arrastre que acepte los datos del arrastrable.",
```

Comment:
A widget that receives data when a Draggable widget is dropped. When a draggable is dragged on top of a drag target, the drag target is asked whether it will accept the data the draggable is carrying. If the user does drop the draggable on top of the drag target (and the drag target has indicated that it will accept the draggable's data), then the drag target is asked to accept the draggable's data.

Un widget que recibe datos cuando un widget Draggable es soltado. Cuando un draggable es soltado encima de un drag target, al drag target se le consultará si acepta los datos que el draggable porta. Si el usuario suelta el draggable encima del drag target (y el drag target ha indicado que aceptará los datos del draggable), entonces el drag target es consultado para aceptar los datos del draggable...


Menudo trabalenguas me sale... me lo parece a mi o esta duplicándose la frase ?

Reply:
pues es que asi esta jajaja ni para donde moverse, haremos lo mejor que podamos jeje